src/libstore/remote-store.cc

```diff
@@ -412,9 +412,16 @@ Path RemoteStore::addToStore(const string & name, const Path & _srcPath,
 
     auto conn(connections->get());
 
+    if (GET_PROTOCOL_MINOR(conn->daemonVersion) < 0x15) {
```

Comment:
That's a pretty tight compatibility requirement :(

Reply:
Right, I honestly don't know whether we could improve the compatibility story here. Ideas are certainly welcome.